subsys/bluetooth/controller/ll_sw/ull_adv_internal.h

```diff
@@ -92,15 +92,16 @@ uint8_t ull_adv_aux_hdr_set_clear(struct ll_adv_set *adv,
 
 /* helper function to calculate common ext adv payload header length */
 static inline uint8_t
-ull_adv_aux_hdr_len_get(struct pdu_adv_com_ext_adv *com_hdr, uint8_t *dptr)
+ull_adv_aux_hdr_len_get(struct pdu_adv_com_ext_adv *com_hdr, uint8_t **dptr)
```

Comment:
Not sure I like this change: Generally I'm of the conviction that a `get` function should never change any data/state of the parameters. I would revert this and increase the pointer outside of the function.